### PR TITLE
Rename login action name and make Edits in the call's function params

### PR DESCRIPTION
**Pull request by masikotimo**
Rename login action name in the token_login function  and remove the replace method of the svc key in the call's params

### CODE REVIEW

File: wialon/api.py [hunk 1]
```diff
@@ -134,7 +134,7 @@ def call(self, action_name, *argc, **kwargs):
             params = json.dumps(kwargs, ensure_ascii=False)
 
         params = {
-            'svc': action_name.replace('_', '/', 1),
```
Comment (Lenka42) [inline wialon/api.py:137]: This replace was made for purpose, not accidentally. This allows to make requests to generic URLs like `core/search_items` by calling method `core_search_items`. Please, do not change this.

File: wialon/api.py [hunk 2]
```diff
@@ -39,7 +39,7 @@ class WialonError(Exception):
         3: 'Invalid result',
         4: 'Invalid input',
         5: 'Error performing request',
-        6: 'Unknow error',
+        6: 'Unknown error',
```
Comment (Lenka42) [inline wialon/api.py:42]: Seems legit :slightly_smiling_face:

**Review by Lenka42 — CHANGES_REQUESTED** [review]

File: wialon/api.py [hunk 3]
```diff
@@ -145,7 +145,7 @@ def call(self, action_name, *argc, **kwargs):
 
     def token_login(self, *args, **kwargs):
         kwargs['appName'] = 'python-wialon'
-        return self.call('token_login', *args, **kwargs)
```
[the remark this answers is not in the document]
Comment (masikotimo) [reply wialon/api.py:148]: i think following with  the documentation the action name is 'token/login'  and all other action names and no longer need for the replace method

File: wialon/api.py (hunk 1 above)
Comment (masikotimo) [reply wialon/api.py:137]: following the remote docs why would there be need to have methods like 'core_search_items' yet they have them like core/search_items

Comment (Lenka42) [reply wialon/api.py:137]: Exactly, the API URL looks like `core/search_items`. But you cannot name Python function `core/search_items`, slash is forbidden in function names. So you call `Wialon().core_search_items(...)`. Here is wrapper https://github.com/wialon/python-wialon/blob/master/wialon/api.py#L205, which passes args to `call` method, which replaces first underscore with slash. This is definitely a feature, not a bug :slightly_smiling_face:

**Pull request closed without merging**